R/help.R

```diff
@@ -89,12 +89,36 @@ help_completion_handler.python.builtin.object <- function(topic, source) {
   if (is.null(doc))
     doc <- ""
   
-  # extract preamble
-  arguments_matches <- regexpr(pattern ='\nArg(s|uments):', doc)
-  if (arguments_matches[[1]] != -1)
-    description <- substring(doc, 1, arguments_matches[[1]])
-  else
-    description <- doc
+  if (is_sphinx_doc(doc)) {
```

Comment:
I think we should check if both this is a Sphinx document and whether `docutils` is available before entering this code path. (In other words, we only attempt Sphinx-style parsing if we know the tools for it are available).

In addition, I wonder if we can use our default help codepaths for Sphinx documentation -- does it at least do something sensible?

Reply:
yes, at least the arguments/auto-completion could get populated with no problem since those are obtained from our Python code instead of from the documentation.